modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/modal/Modal.js

```diff
@@ -24,6 +24,26 @@ import './Modal.scss';
 import delegate from '../delegate/delegate.es';
 import navigate from '../util/navigate.es';
 
+const alert = ({message}) => {
+	openModal({
+		bodyHTML: message,
+		buttons: [
+			{
+				label: Liferay.Language.get('ok'),
+				onClick: ({processClose}) => {
+					processClose();
+				},
+				otherProps: {
+					autoFocus: true,
+				},
+			},
+		],
+		size: 'sm',
+		title: Liferay.Language.get('alert'),
+		withTitle: false,
```

Comment:
`withTitle` was necessary to not have the X button to close the modal, it should be necessary for `window.confirm` PR too but I would validate if it is necessary or not.

/cc @markocikos 

<img width="452" alt="Screen Shot 2022-03-08 at 18 39 09" src="https://user-images.githubusercontent.com/7663513/157329684-6208d050-7ecf-4b06-acde-827045098637.png">

<img width="335" alt="Screen Shot 2022-03-08 at 18 40 09" src="https://user-images.githubusercontent.com/7663513/157329840-acbe480f-42d7-43b4-ae17-2dc7cb4c730f.png">

I added this modal title to be alert but I'm not sure if It will look well than not using it. Also, it required a work on the way that we are passing `title` on Modal component where to make the styles to be the same, I needed to wrap this property on ClayModal.Title component. See below :)

Reply:
> withTitle was necessary to not have the X button to close the modal

It's kinda weird to have `withTitle` prop as `false`, but then still have the title, as you passed in the `title` prop. We should chose either to have title or no. I think that having or not having the X button is not that important. In other words:
- If we want the title, we do not need the `withTitle` prop.
- If we don't want the title, `withTitle` prop is fine, but we should not set the `title` prop

I think either having or not having the title is fine. If you think it looks ugly without title, let's have the title. We can remove it later on, if requirements for it appears.

Reply:
I don't like here that we are using API in a way that it is unclear and unintended. To hide the `x` button, we use combination of `withTitle` set to `false` and set `title`. We are relying on internal implementation specifics, that may change down the line.

If we really really wanted to have an option to hide the `x` button, we should explicitly add this to API. Something like `showCloseIcon` boolean prop.

But I honestly don't see a need do this, I think `x` button is a nice feature. Why go out of our way to remove it? What did id do to deserve this treatment? :cry:

Reply:
> If we really really wanted to have an option to hide the x button, we should explicitly add this to API. Something like showCloseIcon boolean prop.

I think this work should be make on Clay side since it just not hide the close icon but the whole title section, including the title text. For this PR I can creating special naming for it but I think having the same prop naming as Clay does can be easier for developers that used Clay before. 

/cc @matuzalemsteles @julien 

But I can rename it to `showCloseIcon` for avoiding withTitle weird naming

Reply:
I've been following the "conversation/review" thread between you and @markocikos, even though I haven't said anything yet. I agree with Marko on this: I wouldn't hide the `x` button either, but if I really had to, I'd work on making the API a bit more "simpler". This sums it up pretty well:

> To hide the x button, we use combination of withTitle set to false and set title

Additionally ... 

> I think this work should be make on Clay side since it just not hide the close icon but the whole title section, including the title text. For this PR I can creating special naming for it but I think having the same prop naming as Clay does can be easier for developers that used Clay before.

You are right that this should done on the "Clay side", but what I suggest is that you work on the change and submit a pull request. This is obviously just "my opinion", and I don't want to sound rude, but I think that we need to promote more collaboration between the different squads we have in our team. If that's not possible, that's not a problem either: we'll obviously help and work on this with you; but I think that since you have the context it would be easier ... and at the same time, encourage other teams to do the same thing in the (near) future.

In any case, let me know what you think when you can.

Reply:
Well just to give some context here, `withTitle` is an API that will be removed in Clay v4, it was introduced through PR https://github.com/liferay/clay/pull/2979 which is to let the behavior of the Header without the implementation of close and title for example, so it's more of a low-level offer, so I'm not sure if adding an API like `showCloseIcon` will break with all that planning because the close button won't exist anymore in v4 by default. So `withTitle` is more of a temporary property and it does the behavior switch.

For example, this will make it a common default for v4: (You can do this already using `withTitle`)

```jsx
<ClayModal.Header>
  <ClayModal.ItemGroup>
    <ClayModal.Item>
      <ClayModal.TitleSection>
        <ClayModal.Title>
          {'Modal Title'}
        </ClayModal.Title>
      </ClayModal.TitleSection>
    </ClayModal.Item>
    <ClayModal.Item shrink>
      <ClayModal.SubtitleSection>
        <ClayModal.Subtitle>
          {'Modal Subtitle'}
        </ClayModal.Subtitle>
      </ClayModal.SubtitleSection>
    </ClayModal.Item>
  </ClayModal.ItemGroup>
</ClayModal.Header>
```

But we can probably have a component that is high-level to avoid excess markup here, something like:

```jsx
<ClayModal.Header>
  <ClayModal.HeaderWithTitle
    title="Modal Title"
    subtitle="Modal Subtitle"
  />
</ClayModal.Header>
```

Well, this isn't anything definite on how we can simplify the two use cases, so there's aria to work with here yet.

Reply:
> But I honestly don't see a need do this, I think x button is a nice feature. Why go out of our way to remove it? What did id do to deserve this treatment? 😢 

I like X button too but if you take a look on https://github.com/liferay-frontend/liferay-portal/pull/1990/files#r822096160, native dialogs don't have the X button. It should be investigated on https://issues.liferay.com/browse/DESIGN-795

Reply:
@diegonvs I get your point, I guess we just need to make the "API" part nicer, something like `hideCloseButton` or similar.

Reply:
I honestly don't think it's important at all, if we have the `x` icon in the solution or not. It does not affect the main functionality of the solution in any way. It has, in my opinion, an extremely small impact on user experience of an alert. I would be even hesitant to ask Lexicon folks for their opinion, as I think it would mostly waste their time. I would do here whatever is the simplest for us: leave the defaults. If the icon is there by default, leave it in. If the Clay v4 removes it, let it be removed. I would not extend API and complicate thing, unless we need to. We can always do that later down the line, if the product folks decide they really like or dislike the icon.

Reply:
> leave the defaults

Ok, removed `withTitle`, removed `size` and now it's looking like:

```javascript
const openAlertModal = ({message}) => {
	openModal({
		bodyHTML: escapeHTML(message),
		buttons: [
			{
				autoFocus: true,
				label: Liferay.Language.get('ok'),
				onClick: ({processClose}) => {
					processClose();
				},
			},
		]
	});
};
```

Reply:
https://github.com/liferay-frontend/liferay-portal/pull/1990/commits/5087efa8a574dd04e0e7bc473b2869cddec97ce2

Reply:
![crying](https://media.giphy.com/media/vOR92XsXPTOj6/giphy-downsized.gif)